src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/addDriverToCar")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate");
+    private static final String ADD = "/WEB-INF/views/cars/drivers/addDriverToCar.jsp";
```

Comment:
```suggestion
    private static final String ADD = "/WEB-INF/views/cars/drivers/add.jsp";
```
also, use nouns for variables names  (check it everywhere)